cobl2/datatables.py

```diff
@@ -25,7 +25,7 @@ def col_defs(self):
 
 class CoblMeaningCol(LinkCol):
     def order(self):
-        return [Meaning.name, CognateClass.root_language, CognateClass.root_form]
+        return Meaning.name
```

Comment:
With this implementation of `order` and `search`, `CoblMeaningCol` could be replaced by simply passing `model_col=Meaning.name` into `LinkCol` when assembling the column definitions below.